cloudhub-2/modules/ROOT/pages/_partials/create-config.adoc

```diff
@@ -43,10 +41,6 @@ You cannot use the following reserved CIDR blocks:
 169.254.0.0/16
 127.0.0.0/8
 0.0.0.0/8
-100.64.0.0/16
```

Comment:
can we restore the change here? they are not related to reserved corporate cidrs feature.